manticore/utils/helpers.py

```diff
@@ -75,33 +79,45 @@ def deserialize(self, f):
         raise NotImplementedError
 
 
-class PickleSerializer(StateSerializer):
+class GzipPickleSerializer(StateSerializer):
```

Comment:
As long as we're not creating side-by-side `PickleSerializer` and `GZipPickleSerializer` options, why not leave the name the same so that the diff will be smaller?

Reply:
If you know of a way to have Github Actions to test each commit in a PR and every commit passed, we could merge this PR without squashing and review each commit individually.

Here's the commit for the transparent `PickleSerializer` change without renmaing https://github.com/trailofbits/manticore/pull/1624/commits/4409c00c91f8c6573b3fe0638be31858f12f026b

Reply:
Sure, I'm fine with leaving `PickleSerializer` named as-is. :)